ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -291,6 +301,13 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
     });
   });
 
+  const solanaAccountCreated = useMemo(() => {
```

Comment:
`solanaAccountCreated` doesn't scream to me what it does.  Does this determine if a user has any Solana accounts?  Maybe `userHasSolanaAccount`?